Testing conda moose creation on intel macs

milljm opened the pull request:
I can build manually (sometimes), civet can build as well (sometimes).
Committing this for one more test case.

Refs #0000

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212160/) on 43b5d7d wanted to post the following:

A change of the following file(s) triggered this check:

  conda/petsc

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212160/) on 43b5d7d wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212160/) on 43b5d7d wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212235/) on 2d35622 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/petsc

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212235/) on 2d35622 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1212235/) on 2d35622 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Add empty Apptainer definition files

Add empty Apptainer recipe/definition files so that module_hash
functions correctly.

Closes #22505

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1214668/) on 7d02d01 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/mpich

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1214953/) on 1df9cfa wanted to post the following:

A change of the following file(s) triggered this check:

  conda/mpich

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1215098/) on 24847e0 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
Try a libmesh bump to build moose

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1215150/) on d4bc7e3 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/libmesh

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1215150/) on d4bc7e3 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1215150/) on d4bc7e3 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/libmesh

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1215150/) on d4bc7e3 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

cticenhour commented:
@milljm This testing PR seems unnecessary now, and the stalebot will get it soon anyway, so I am closing it.
